src/test/scala/chiselTests/ChiselSpec.scala

```diff
@@ -124,38 +126,50 @@ trait ChiselRunners extends Assertions {
     assert(!runTester(t, additionalVResources, annotations))
   }
 
-  def assertKnownWidth(expected: Int)(gen: => Data): Unit = {
+  def assertKnownWidth(expected: Int, args: Iterable[String] = Nil)(gen: => Data)(implicit pos: Position): Unit = {
```

Comment:
Huh, so this is for scalatest to properly pass the source locator to the assert? Interesting.